libmscore/chord.cpp

```diff
@@ -2285,6 +2285,8 @@ void Chord::layoutTablature()
                         || prevCR->tuplet() != tuplet()
                         || prevCR->type() == ElementType::REST)
                         needTabDur = true;
+                  else if (tab->symRepeat() == TablatureSymbolRepeat::NEVER)
+                        ; // No tab duration symbol needed
```

Comment:
This code will lead to a warning on most compilers. Can the code be rearranged to avoid empty `if` case?